packages/RatPack.package/RPTests.class/instance/testTokenValidation.st

```diff
@@ -0,0 +1,17 @@
+test - session middleware
+testTokenValidation
```

Comment:
Validation is not really correct here, since this tests more than just the signature validation. It tests the whole parsing process. I can't come up with a better name right now, though.

Reply:
Is the name of the test the only thing what is still to do in the PR?

Reply:
yes

Reply:
I think there is no more value by the word process, so I'd suggest: testTokenParsing.
Maybe you also could rename the instanceVar: session to sessionMiddleware ?!

Reply:
the method rename should be better done within squeak?

Reply:
yes, would be better

Reply:
can anybody do that, please? is @He3lixxx ok with the naming now?